first-network/byfn.sh

```diff
@@ -153,6 +153,7 @@ function networkUp() {
   # generate artifacts if they don't exist
   if [ ! -d "crypto-config" ]; then
     generateCerts
+    replacePrivateKey
```

Comment:
This looks like a merge failure, as this line was recently removed - please can you undo this change?

Reply:
Sure.